### DISCUSSION TITLE
Display a Home Assistant lovelace Card in HA-floorplan

### DISCUSSION THREAD

**Discussion by lucibuz** [ExperienceLovelace/ha-floorplan#163] [Q&A]
How can be display a lovelace card (alarm-panel or weather forecast card) in ha-floorplan, similar as camera image?

**Comment by OzGav** [comment]
I don’t believe you can directly but maybe sidebar view will do what you need https://www.home-assistant.io/lovelace/sidebar/

**Reply by lucibuz** [reply]
Thanks for the info, I'll check it out. But I don't believe sidebar will display the card on top of the ha-floorplan .
I use to achieve this having multiple cards including floorplan display as elements in picture-elements, with proper styling.
Now I'm looking for a simple way ..
![UIweather](https://user-images.githubusercontent.com/53460444/141746948-ebd5d633-edbe-48d9-aaa4-03cf91ab5c47.png)
.

**Comment by OzGav** [comment]
Correct my suggestion will only show other cards beside ha-floorplan. Help with picture elements cards is beyond the scope of this discussion as we are just here to help with ha-floorplan. ha-floorplan can’t display other cards on top at this time. 

Some of what you have in your image can be done easily as the pictures are not dynamic. That is, all of the weather information except the graph.

If you could save the camera image locally then you could pull that into floorplan. I see there are some other threads on the forums about saving camera images. 

So the graph I don’t think I can help with except to suggest again maybe sideview for that?

**Reply by lucibuz** [reply]
Thanks for the feedback.
I'll have a look at the sideview option and get back.
